docs/website/docs/getting-started/use-an-existing-source.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Use an existing source
+description: Use an existing source - how to leverage what others did before you
+keywords: [Use an existing a data source]
+---
+
+# How to use an existing verified source
+
+While dlt is primarily a pipeline building tool, on dlthub we host some common pipelines and also encourage the community to build and distribute high quality pipelines.
+Read on.
+
+## What is not a verified source but still widely available?
```

Comment:
I'd explain what verified source is first before saying what it is not :) tbh. drop this section, it does not add much